integration-tests/telegram/pom.xml

```diff
@@ -56,17 +56,18 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-attachments</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.camel.quarkus</groupId>
-            <artifactId>camel-quarkus-integration-test-support-mock-backend</artifactId>
-        </dependency>
```

Comment:
The auditing idea was based on the assumption that the number of <artifactId>camel-quarkus-integration-test-support-mock-backend</artifactId> occurrences in poms must be equal to the number of the magic log tokens. Now we are stopping to use <artifactId>camel-quarkus-integration-test-support-mock-backend</artifactId> . 

Yes, <artifactId>camel-quarkus-integration-wiremock-support</artifactId> is a replacement, but probably not all mock will be HTTP, so the interested parties (cc @llowinge ) will have to add the number of <artifactId>camel-quarkus-integration-wiremock-support</artifactId> occurrences to the number of <artifactId>camel-quarkus-integration-test-support-mock-backend</artifactId>.

Reply:
Let me think about this some more because there are probably other (better) ways of achieving this.